_ecosystem/OpenMMLab

```diff
@@ -0,0 +1,10 @@
+---
+layout: ecosystem_detail
+title: OpenMMLab
+summary: OpenMMLab covers a wide range of research topics of computer vision, e.g., classification, detection, segmentation and super-resolution.
+link: https://github.com/open-mmlab
+summary-home: OpenMMLab covers a wide range of research topics of computer vision, e.g., classification, detection, segmentation and super-resolution.
```

Comment:
Would change the wording slightly:

"covers a wide range of computer vision research topics including classification, detection, segmentation, and super-resolution."